iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -324,7 +324,8 @@ public override async Task EnableTwinPatchAsync(CancellationToken cancellationTo
             try
             {
                 cancellationToken.ThrowIfCancellationRequested();
-                await _amqpUnit.SendTwinMessageAsync(AmqpTwinMessageType.Put, Guid.NewGuid().ToString(), null, _operationTimeout).ConfigureAwait(false);
+                string correlationId = AmqpTwinMessageType.Put + Guid.NewGuid().ToString();
```

Comment:
Adding "Put/Patch/Get" before correlation Id to be able to identify response.

Reply:
I don't really see a need of that but thanks for the suggestion

Reply:
I don't like the idea of making it that much longer. Can we overlay it on top of the first group?

Reply:
Can you elaborate @drwill-ms ?

I am concerned about the uniqueness of correlation Id if we shorten it so would like to understand what you meant.

Reply:
You do not need to worry about the uniqueness of guids, even if you clip the first 5 characters with this small of a data set. Do you realize how large and unique a number a guid is?

Reply:
Yes I do.

So what we need here is 
for example guid is - abcd12345
for Get we want it to be - **get**d12345

Is that correct?

Reply:
yep